osu.Game/Overlays/Settings/Sections/Maintenance/MassVideoDeleteConfirmationDialog.cs

```diff
@@ -10,7 +11,7 @@ public class MassVideoDeleteConfirmationDialog : MassDeleteConfirmationDialog
         public MassVideoDeleteConfirmationDialog(Action deleteAction)
             : base(deleteAction)
         {
-            BodyText = "All beatmap videos? This cannot be undone!";
+            BodyText = MaintenanceSettingsStrings.MassVideoDeleteConfirmation;
```

Comment:
This isn't going to localise well. "Delete" needs to be in the string. Probably remove for now.

Reply:
Okay